### PR TITLE
Updates and additions for new Substrate tx_relayer handlers

### PR DESCRIPTION
**Pull request by drewstone**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
- [ ] Yes
- [ ] No

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -41,7 +41,7 @@ hex = { version = "0.4", default-features = false }
 # just to make it compile on linux where the openssl is not available.
 # until ethers-rs solve this issue: https://github.com/gakonst/ethers-rs/issues/325
 native-tls = { version = "^0.2", features = ["vendored"] }
-webb = { version = "0.3.0", default-features = false }
+webb = { version = "0.3.2", default-features = false }
```
Comment (dutterbutter) [inline Cargo.toml:44]: I think this should `0.3.3`. See https://github.com/webb-tools/relayer/pull/156/files#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542R44

File: src/tx_relay/evm/mod.rs [hunk 2]
```diff
@@ -1,2 +1,79 @@
...
+};
+
+use crate::handler::{
+    into_withdraw_error, CommandResponse, CommandStream, WithdrawStatus,
+};
+
 pub mod anchor;
 pub mod vanchor;
+
+pub async fn handle_evm_tx<M, D>(
```
Comment (dutterbutter) [inline src/tx_relay/evm/mod.rs:15]: Add documentation here please.

File: src/tx_relay/mod.rs [hunk 3]
```diff
@@ -1,2 +1,90 @@
...
+    /// The refund for the transaction in native tokens
+    pub refund: B,
+    /// The refresh commitment
+    pub refresh_commitment: E,
+    /// The external data hash,
+    pub ext_data_hash: E,
+}
+
+#[derive(Debug, Clone, Deserialize, Serialize)]
+pub struct ProofData<P, R, E> {
```
Comment (dutterbutter) [inline src/tx_relay/mod.rs:59]: Please add documentation.

File: src/tx_relay/mod.rs [hunk 4]
```diff
@@ -1,2 +1,90 @@
...
+    pub proof: P,
+    pub public_amount: E,
+    pub roots: R,
+    pub input_nullifiers: Vec<E>,
+    pub output_commitments: Vec<E>,
+    pub ext_data_hash: E,
+}
+
+#[derive(Debug, Clone, Deserialize, Serialize)]
+pub struct ExtData<E, I, B, A> {
```
Comment (dutterbutter) [inline src/tx_relay/mod.rs:69]: Please add documentation.

File: src/tx_relay/substrate/mod.rs [hunk 5]
```diff
@@ -1,3 +1,93 @@
...
+    DefaultConfig, TransactionProgress, TransactionStatus,
+};
+
+use crate::handler::{CommandResponse, CommandStream, WithdrawStatus};
+
 pub mod anchor;
 pub mod mixer;
 pub mod vanchor;
+
+pub async fn handle_substrate_tx<E: Decode + Debug>(
```
Comment (dutterbutter) [inline src/tx_relay/substrate/mod.rs:16]: Please add documentation.

**Review by dharjeezy — APPROVED** [review]

**Pull request merged**